peachjam/js/peachjam.ts

```diff
@@ -66,6 +66,7 @@ class PeachJam {
   }
 
   setupHtmx () {
+    // @ts-ignore
```

Comment:
@longhotsummer everything works okay except now this line throws a typescript error on compilation